infrastructure/modules/keyvault/main.tf

```diff
@@ -0,0 +1,71 @@
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_key_vault" "secrets_vault" {
+  name                = var.name
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  soft_delete_enabled = true
+
+  sku_name = "standard"
+
+  network_acls {
+    default_action = var.network_acls_default_action
+    bypass         = "AzureServices"
+    ip_rules       = var.access_CIDR
+  }
+
+  tags = {
+    environment = var.environment
+  }
+
+  dynamic "access_policy" {
```

Comment:
This access policy seems to be granting every permission to every authorised person.

Do we have an access policy for reading only, or maybe for writing only?

Reply:
This is the current list of permissions assigned to each of us currently, I didn't see any reason to lock it down further for each of us.